autowarefoundation/autoware
# [Feature] velodyne drivers update

## Status
**PRODUCTION / DEVELOPMENT**

## Description
Updated and fixed subtree related to the Velodyne drivers fork since the initial Autoware release.
autowarefoundation/autoware_ai#56


## Todos
  - [X] Ubuntu 14.04, ROS Indigo
  - [x] Ubuntu 16.04, ROS Kinetic
- Tested with real sensors
  - [X] Velodyne VLP-16
  - [X] Velodyne VLP-16 Hi-Res
- Tested with rosbag VelodynePackets
  - [X] Velodyne HDL-64E S2
  - [X] Velodyne HDL-64E S3
  - [X] Velodyne HDL-32E

1. Compile the branch
```
./catkin_make_release
```
1. Launch Runtime Manager
1. Play a Rosbag with VelodynePackets or connect a real Velodyne scanner
1. Go to Sensing tab and launch the respective driver to your sensor.
1. Go to Rviz and add `/points_raw` topic

PointCloud should be displayed
